llm_council/processors/services/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .base_service import BaseService
+from .anthropic_service import AnthropicService
+from .cohere_service import CohereService
```

Comment:
Were there more services that you want to add here?

Reply:
definitely, they should all be in here. Not sure what happened.